### ISSUE TITLE
No intellisense in VSCode

### ISSUE THREAD

**Issue by bearrito** [youtype/mypy_boto3_builder#176]
**Describe the bug**
Not seeing any intellisense or type checking in Vscode

**To Reproduce**

1. Install vscode extensions
2. Add the services you want

**Actual output**

I am able to see method signatures, and links to documentation, but I don't seem to get any type checking.
For instance the below doesn't show any warning/intellisense. In this case, name is wrong type and not_field does not exist.

```
   self.table = resource.Table(name=1, not_field=True)
```

My VScode is as follows

```
Version: 1.71.2
Commit: 74b1f979648cc44d385a2286793c226e611f59e7
Date: 2022-09-14T21:12:14.256Z
Electron: 19.0.12
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Linux x64 5.15.0-53-generic
Sandboxed: No
```

**Comment by vemel** [comment]
Please try to update. It could be due to invalid imports in `mypy-boto3-cloudformation` package. Make sure that you have `mypy-boto3-cloudformation 1.26.35.post1` installed.

**Comment by rdbisme** [comment]
Hey I have the same problem: 

```python
for bucket in buckets:
    async with session.resource(
        "s3", endpoint_url=self._s3_endpoint_url, region_name=bucket["region"]
    ) as s3_resource:
        reveal_type(s3_resource)
        bkt = await s3_resource.Bucket(bucket["name"])
```

Here `reveal_type` provided by PyLance hovering the mous on it is `Any`, and it should be `S3ServiceResource`.

Hovering the mous over the `.resource` method of `session`, I see the right signature with the correct typing. I have installed with `poetry add -D types-aioboto3[s3]`.

**Comment by rdbisme** [comment]
Hey, I have the same problem:

```python
for bucket in buckets:
    async with session.resource(
        "s3", endpoint_url=self._s3_endpoint_url, region_name=bucket["region"]
    ) as s3_resource:
        reveal_type(s3_resource)
        bkt = await s3_resource.Bucket(bucket["name"])
```

Here reveal_type provided by PyLance hovering the mouse on it is `Any`, and it should be `S3ServiceResource`

Hovering the mouse over the `.resource` method of session, I see the right signature with the correct typing. I have installed with poetry add -D types-aioboto3[s3].

**Comment by bearrito** [comment]
Here is my pip list

```
mypy                                 0.761
mypy-boto3-cloudformation            1.26.35.post1
mypy-boto3-dynamodb                  1.26.24
mypy-boto3-ec2                       1.26.34
mypy-boto3-ecs                       1.24.90
mypy-boto3-efs                       1.24.36.post1
mypy-boto3-lambda                    1.26.18
mypy-boto3-rds                       1.26.36
mypy-boto3-s3                        1.26.0.post1
mypy-boto3-sqs                       1.26.0.post1
mypy-extensions                      0.4.3
```

**Comment by vemel** [comment]
@bearrito do you have `boto3-stubs` installed as well?

**Comment by bearrito** [comment]
@vemel  

```
boto3                                1.26.44
boto3-stubs                          1.26.44
botocore                             1.29.44
botocore-stubs                       1.29.44
```

**Comment by bearrito** [comment]
Closing. Installing on a new workstation didn't show any use cases.
